nyaa/forms.py

```diff
@@ -123,6 +123,13 @@ def pre_validate(self, form):
             raise ValueError(self.gettext('Not a valid choice'))
 
 
+class CommentForm(FlaskForm):
+    comment = TextAreaField('Make a comment', [
+        Length(max=10 * 1024, message='Comment must be at most %(max)d characters long.'),
```

Comment:
Do we really want comments this big?
I think 255 is plenty.

It's not like people do anime reviews in the comment section, or do they?